src/pages/components/input-group.mdx

```diff
@@ -95,8 +95,8 @@ The InputGroup component follows this structure:
 
 ```jsx
 <Field>
-  <Label />
-  <Hint />
+  <Field.Label />
+  <Field.Hint />
   <InputGroup />
 </Field>
 ```
```

Comment:
Should the `InputGroup` page display `Field`'s APIs?

Reply:
I think there was a concern about `Field` taking API prominence. We can probably leave it off here.